master/internal/trial.go

```diff
@@ -367,7 +385,7 @@ func (t *trial) addTask() error {
 	return t.db.AddTask(&model.Task{
 		TaskID:     t.taskID,
 		TaskType:   model.TaskTypeTrial,
-		StartTime:  t.jobSubmissionTime,
+		StartTime:  t.jobSubmissionTime, // TODO: Why is this the job submission time..?
```

Comment:
iirc we do a similar thing with commands, not saying this behavior is correct because we do it in two places

Reply:
ehhh. yeah. task start time is just.. when the task was requested. so it really should be user submission time for commands and searcher request time for experiments. imo.